.github/workflows/main.yml

```diff
@@ -291,6 +291,57 @@ jobs:
             fi
           fi
 
+  # This job deploys snapshots on the master branch
+  DeployJavaSnapshot:
+    needs: [BuildJMonkey]
+    name: Deploy Java Snapshot
+    runs-on: ubuntu-latest
+    if: github.ref_name == 'master'
```

Comment:
Will this job be gnored on pull requests to master branch? 
because that is what we want. It should only run on commit push.

Reply:
probably we want 
```
if: github.event_name == 'push' && github.ref_name == 'master'
```
right?

Reply:
Probably. I knew I was losing track of something here.